sdk/python/feast/job_service.py

```diff
@@ -145,6 +167,54 @@ def GetJob(self, request, context):
         return GetJobResponse(job=self._job_to_proto(job))
 
 
+def run_control_loop():
+    """Runs control loop that continuously ensures that correct jobs are being run."""
```

Comment:
I think we need to be more explicit about which `jobs` are being run, and what the control loop will do (killing, restarting)

Reply:
got it